coriolisclient/cli/endpoint_instances.py

```diff
@@ -40,6 +43,35 @@ def _get_formatted_data(self, obj):
         return data
 
 
+class InstancesDetailFormatter(formatter.EntityFormatter):
+
+    def __init__(self, show_instances_data=False):
+        self.columns = [
+            "ID",
+            "Name",
+            "Flavor",
+            "Memory MB",
+            "Cores",
+            "OS Type",
+            "Devices",
+            "Firmware type"
+        ]
+
+    def _get_formatted_data(self, obj):
+        data = [
+            obj.id,
+            obj.name,
```

Comment:
We should ideally print the 'instance_name', as that is the full instance identifier (ex: on vSphere 'instance_name' would be "CI/test/test2/vm" while 'name' would just be 'vm'.

Reply:
instance name is presented by all providers?

Reply:
FFS my comment got lost.
Yes, it should be present in all export providers.
I see that it's not in the core JSON schemas, which it absolutely should.